packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.ts

```diff
@@ -326,6 +326,7 @@ export default ({ api, coreSagas, networks }) => {
         // store initial address in case of US state we add prefix
         const userState = country === 'US' ? `US-${state}` : state
         yield call(api.setUserInitialAddress, country, userState)
+        yield call(coreSagas.settings.fetchSettings)
```

Comment:
originally it was added to be fetched for first time login on where we do set currency, right after that, I moved it here, because if it stay on that line it will fetch non updated state and currency will remain as default USD